node/node_explorer.go

```diff
@@ -81,6 +82,13 @@ func (node *Node) ExplorerMessageHandler(payload []byte) {
 	return
 }
 
+// AddNewBlockForExplorer add new block for explorer.
+func (node *Node) AddNewBlockForExplorer(block *types.Block) {
+	// TODO: Currently assumping blocks come in order.
```

Comment:
what's the impact if blocks didn't come in order?

Reply:
I'm writing another PR for this.
1/ i don't want to have a big PR
2/ i was telling another PR will come right after that - that's why i need to check in this one first.